addon/ActivityStreams.js

```diff
@@ -226,6 +226,8 @@ ActivityStreams.prototype = {
     switch (msg.type) {
       case am.type("NOTIFY_BOOKMARK_ADD"):
         PlacesProvider.links.asyncAddBookmark(msg.data);
+        // Request metadata for the new bookmark.
+        this._pageScraper.asyncFetchLinks([{url: msg.data}], msg.type);
```

Comment:
are you able to check if the bookmark request came from a pocket story and then if it did then do this? i would rather not do a db lookup if we don't have to